src/game/default/g_main.c

```diff
@@ -796,21 +799,23 @@ static void G_Frame(void) {
 			return;
 		}
 	}
+		
+	if (!TIMEOUT) {
```

Comment:
`g_level.time` does not stop. Items will continue to rotate and bob during a timeout, and things like water will continue to do whatever it does. From what I've tested, things like lifts, projectiles and temp entities all lock in place and continue where they were upon timing in. Even the grinder in The Pits just stops where it is and then continues on.

Instead of stopping `g_level.time` I just set `g_level.frame_num` back to where it was when timeout was called. Maybe this will screw up something in the back end that I didn't think of, but it seems to work just fine.